charts/argocd-apps/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: argocd
```

Comment:
Might be worth thinking about the naming of this and the directory, could maybe just go with `apps`?